conf/composites.conf

```diff
@@ -53,7 +53,17 @@ composites {
     expression = "R_DKIM_NA & R_SPF_NA & DMARC_NA & ARC_NA";
     score = 1.0;
     policy = "remove_weight";
-    description = "Authenticating message via SPF/DKIM/DMARC/ARC not possible";
+    description = "Authenticating message via SPF/DKIM/DMARC/ARC not available";
+  }
+  AUTH_NA_OR_FAIL {
+    expression = "!(R_DKIM_NA & R_SPF_NA & DMARC_NA & ARC_NA) & (R_DKIM_NA | R_DKIM_TEMPFAIL | R_DKIM_PERMFAIL) & (R_SPF_NA | R_SPF_DNSFAIL) & DMARC_NA & (ARC_NA | ARC_DNSFAIL)";
+    score = 1.0;
+    policy = "remove_weight";
+    description = "Authenticating message via SPF/DKIM/DMARC/ARC not available or failed";
```

Comment:
This message is not very descriptive to me. What is `via` here, for example? SPF/DMARC are operating on different definitions of the sender's address...

Reply:
I copied this description from `AUTH_NA` and added `or failed`.